flashcard_runner.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/card'
+require './lib/deck'
+require './lib/turn'
+require './lib/round'
+require './lib/card_generator'
+
+def start
+  filename = "./lib/cards.txt"
+  cards = CardGenerator.new(filename).cards
+  deck = Deck.new(cards)
+  round = Round.new(deck)
+
+  puts "Welcome! You're playing with #{deck.count} cards"
+  i = 1
+  deck.cards.each do |card|
```

Comment:
Ruby arrays have a built in `each_with_index` method you might find handy here 😉

Reply:
Ahhh I love this! never fails to amaze me how many enumerable are available in ruby